gravitee-apim-console-webui/src/management/api/endpoints-v4/endpoint-group/api-endpoint-group.component.ts

```diff
@@ -126,9 +136,44 @@ export class ApiEndpointGroupComponent implements OnInit, OnDestroy {
       }),
     });
 
+    this.healthCheckForm = new FormGroup({
+      enabled: new FormControl({
+        value: this.endpointGroup.services.healthCheck?.enabled ?? false,
+        disabled: this.isReadOnly,
+      }),
+      configuration: new FormControl(
+        {
+          value: this.endpointGroup.services.healthCheck?.configuration ?? {},
+          disabled: this.isReadOnly,
+        } ?? {},
+      ),
+    });
+
+    this.apiServicePluginsV2Service
+      .getApiServicePluginSchema(ApiHealthCheckV4FormComponent.HTTP_HEALTH_CHECK)
+      .pipe(
+        tap((schema) => {
+          this.healthCheckSchema = schema;
+
+          this.healthCheckForm
+            .get('enabled')
+            .valueChanges.pipe(startWith(this.healthCheckForm.get('enabled').value), takeUntil(this.unsubscribe$))
+            .subscribe((enabled) => {
+              if (enabled) {
+                this.healthCheckForm.get('configuration').enable({ emitEvent: false });
```

Comment:
As the form is typed now I would do 

```suggestion
                this.healthCheckForm.controls.configuration.enable({ emitEvent: false });
```

( same for the disable )